services/ux-backend/handlers/onboarding_tokens.go

```diff
@@ -0,0 +1,131 @@
+package handler
+
+import (
+	"crypto"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/base64"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/red-hat-storage/ocs-operator/v4/services/types"
+	"k8s.io/klog/v2"
+)
+
+func OnboardingTokensHandler(w http.ResponseWriter, r *http.Request) {
+
+	var err error
+	switch r.Method {
+	case "POST":
+		onboardingToken, err := generateOnboardingToken()
```

Comment:
An init function w/ pkg level var might fit the purpose.

Reply:
Right now i have added in the main.go, that is good enough i feel

Reply:
@leelavg @rchikatw 
An init function should be used only in extreme cases where you need to guarantee code loaded at package reference time.  It adds a lot of ambiguity to the order of execution and might change it just because someone had referenced a package something in another module. It is a bad practice to use it unless you are trying to implement something like dependency injection mechanism. 

To add to the above, we are in the main module, there will be no difference between init and just having the code in main (which is always invoked first). 

Let's remove the complexity and have everything back in main.